### PR TITLE
Turn on Cql Timestamp to JodaTime DateTime codec

### PR DESCRIPTION
**Pull request by hochung**
Setup codec for Cql Timestamp, Date, Time to JodaTime DateTime, Local Date, LocalTime

### CODE REVIEW

File: src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        } catch (IllegalArgumentException e) {
+            throw new InvalidTypeException(String.format("Cannot parse timestamp value from \"%s\"", value));
+        }
+    }
+
+    @Override
+    public String format(DateTime value) {
+        if (value == null)
+            return "NULL";
+        return Long.toString(value.getMillis());
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java]: What's the reason for not using the DateTime representation of a string?

File: src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java [hunk 2]
```diff
@@ -75,12 +90,21 @@ public static Entity mapRowToEntity( Row row, Set<PropertyType> properties ) {
 	public static SetMultimap<FullQualifiedName, Object> mapRowToObject( Row row, Collection<PropertyType> properties ) {
 		SetMultimap<FullQualifiedName, Object> map = HashMultimap.create();
 		properties.forEach(property -> {
-			Object value = row.getObject( "value_" + property.getTypename() );
+			Object value = getObject( row, CassandraEdmMapping.getCassandraType( property.getDatatype() ), "value_" + property.getTypename() );
```
Comment (geekbeast) [inline src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java]: This probably legacy badness from way back-- do you remember why we're prefixing `value_`?

File: src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java (hunk 1 above)
Comment (hochung) [inline src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java]: TypeCodec.format is supposed to format the JavaType as a CQL string. TypeCodec.deserialize handles what you are thinking about I guess.

File: src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java (hunk 2 above)
Comment (hochung) [inline src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java]: We probably can revamp this a little bit - the main reason is that when @Bloodyhacker creates the cache table, he prefixes the column names by `value_`
https://github.com/dataloom/kindling/blob/develop/src/main/java/com/kryptnostic/sparks/ConductorSparkImpl.java#L372

File: src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java (hunk 1 above)
Comment (geekbeast) [inline src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java]: I'm not really sure when format gets called here as it is supposed to prefer ByteBuffer serialization, but based off the docs it looks like the CQL string literal it has to be in ISO 8601 format. It does say you can use an integer for input, but I think that's means actual integer not string. What do other codecs do?

Comment (geekbeast) [inline src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java]: `InstantCodec` uses  `java.time.format.DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSxxx")`

File: src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java (hunk 2 above)
Comment (geekbeast) [inline src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java]: Ah okay

File: src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java (hunk 1 above)
Comment (hochung) [inline src/main/java/com/kryptnostic/conductor/codecs/TimestampDateTimeTypeCodec.java]: Hmm good catch. Apparently format is called in these occasions:
http://docs.datastax.com/en/drivers/java/3.1/com/datastax/driver/core/TypeCodec.html#format-T-
Anyway I updated the format function.

**Pull request merged**